app/controllers/cases_controller.rb

```diff
@@ -21,7 +23,15 @@ class CasesController < ApplicationController
                   :process_date_responded,
                   :remove_pit_extension,
                   :record_late_team,
-                  :update_closure
+                  :update_closure,
+                  :response_upload_and_approve,
+                  :response_upload_for_redraft,
+                  :upload_responses,
+                  :upload_responses_action,
+                  :upload_response_and_approve,
+                  :execute_upload_response_and_approve,
+                  :upload_response_and_return_for_redraft,
+                  :upload_response_and_return_for_redraft_action,
```

Comment:
Should this action name now be `execute_upload_response_and_return_for_redraft` or is it a special case of an action? Which is more sensible in terms of the existing code base ?